packages/nimble-components/src/table/specs/table-column-specs/table-column-text-field.md

```diff
@@ -68,8 +68,8 @@ public class TableColumnText extends TableColumn<TableColumnTextCellRecord, Tabl
 
```

Comment:
Should also update the top-level table-columns-hld (since that's kinda our only docs right now): https://github.com/ni/nimble/blob/f6d3c1d10467c3cf640afaf743fb0fdb4c46b6d6/packages/nimble-components/src/table/specs/table-columns-hld.md#tablecolumn